### PR TITLE
Databases-week1 homework

### PR DESCRIPTION
**Pull request by na-talia**

### CODE REVIEW

File: databases/week1/index.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ORDER BY created DESC;
+
+/* Get the single most recently created task */
+SELECT *
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+/* Get the title and due date of all tasks where the title or description contains database */
+SELECT task.title, task.due_date
```
Comment (dboyadzhiev) [inline databases/week1/index.sql]: the table name is not necessary here, it is not wrong though.
```suggestion
SELECT title, due_date 
```

File: databases/week1/index.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+OR title LIKE "%database%";
+
+/* Get the title and status (as text) of all tasks */
+SELECT task.title, status.name 
+FROM task 
+JOIN status ON status.id = task.status_id;
+
+/* Get the name of each status, along with a count of how many tasks have that status */
+SELECT status.name, 
+COUNT(*) as number
```
Comment (dboyadzhiev) [inline databases/week1/index.sql]: it is better to specify which table data you want to count
```suggestion
COUNT(task.id) as number 
```

**Review by dboyadzhiev — APPROVED** [review]

**Comment by dboyadzhiev** [comment]
Good work 👍

**Comment by na-talia** [comment]
> Good work 👍

Thank you!

**Pull request merged**